iotdb-core/metrics/core/src/main/java/org/apache/iotdb/metrics/core/IoTDBMetricManager.java

```diff
@@ -93,7 +93,7 @@ public Gauge createGauge() {
   }
 
   @Override
-  public Histogram createHistogram(MetricInfo metricInfo) {
+  public Histogram createHistogram() {
```

Comment:
This may cause inconsistency, maybe we need to pass metricInfl

Reply:
Why? It is not used finally for now, see https://github.com/apache/iotdb/blob/master/iotdb-core/metrics/core/src/main/java/org/apache/iotdb/metrics/core/IoTDBMetricManager.java#L96